library/junos_commit

```diff
@@ -142,6 +150,10 @@ def main():
         # --- UNREACHABLE ---
 
     try:
+        timeout = int(args['timeout'])
+        if timeout > 0:
+            dev.timeout = timeout
+
```

Comment:
Thanks @stacywsmith 
Can you check line https://github.com/Juniper/ansible-junos-stdlib/blob/master/library/junos_install_os#L234
to predefine timeout data type. with this code would change to 
```timeout = intargs['timeout']```